src/pages/settings/Profile/ProfilePage.js

```diff
@@ -66,20 +66,19 @@ class ProfilePage extends Component {
     constructor(props) {
         super(props);
 
-        this.defaultAvatar = ReportUtils.getDefaultAvatar(this.props.currentUserPersonalDetails.login);
-
+        const currentUserDetails = this.props.currentUserPersonalDetails || {};
         this.state = {
-            firstName: this.props.currentUserPersonalDetails.firstName,
+            firstName: currentUserDetails.firstName || '',
             hasFirstNameError: false,
-            lastName: this.props.currentUserPersonalDetails.lastName,
+            lastName: currentUserDetails.lastName || '',
             hasLastNameError: false,
-            pronouns: this.props.currentUserPersonalDetails.pronouns,
+            pronouns: currentUserDetails.pronouns || '',
             hasPronounError: false,
-            hasSelfSelectedPronouns: !_.isEmpty(this.props.currentUserPersonalDetails.pronouns) && !this.props.currentUserPersonalDetails.pronouns.startsWith(CONST.PRONOUNS.PREFIX),
-            selectedTimezone: lodashGet(this.props.currentUserPersonalDetails, 'timezone.selected', CONST.DEFAULT_TIME_ZONE.selected),
-            isAutomaticTimezone: lodashGet(this.props.currentUserPersonalDetails, 'timezone.automatic', CONST.DEFAULT_TIME_ZONE.automatic),
+            hasSelfSelectedPronouns: !(currentUserDetails.pronouns || '').startsWith(CONST.PRONOUNS.PREFIX),
```

Comment:
@Beamanator I think this change is not gonna result in the same boolean right? 

if pronouns is empty, we will get false for `hasSelfSelectedPronouns ` here:

`!_.isEmpty(this.props.currentUserPersonalDetails.pronouns)`

but if pronouns is empty with the new line, that would be startsWith(PREFIX) is false and negating the entire line will give `true`.

Nonetheless, even if it is correct, it is hard to understand 😄

Reply:
Dude good shout, this would probably cause weird behavior if `pronouns` is empty 🙃 👍 

I can revert part of the refactor here, just keeping the `currentPersonalDetails` simplification

Reply:
@Beamanator Sounds good!